collectors/daemonset_test.go

```diff
@@ -88,12 +94,18 @@ func TestDaemonSetCollector(t *testing.T) {
 				kube_daemonset_metadata_generation{namespace="ns2",daemonset="ds2"} 14
 				kube_daemonset_status_current_number_scheduled{namespace="ns1",daemonset="ds1"} 15
 				kube_daemonset_status_current_number_scheduled{namespace="ns2",daemonset="ds2"} 10
-				kube_daemonset_status_number_misscheduled{namespace="ns1",daemonset="ds1"} 10
-				kube_daemonset_status_number_misscheduled{namespace="ns2",daemonset="ds2"} 5
 				kube_daemonset_status_desired_number_scheduled{namespace="ns1",daemonset="ds1"} 5
 				kube_daemonset_status_desired_number_scheduled{namespace="ns2",daemonset="ds2"} 0
+				kube_daemonset_status_number_available{daemonset="ds1",namespace="ns1"} 0
+		    kube_daemonset_status_number_available{daemonset="ds2",namespace="ns2"} 0
+				kube_daemonset_status_number_misscheduled{namespace="ns1",daemonset="ds1"} 10
+				kube_daemonset_status_number_misscheduled{namespace="ns2",daemonset="ds2"} 5
 				kube_daemonset_status_number_ready{namespace="ns1",daemonset="ds1"} 5
 				kube_daemonset_status_number_ready{namespace="ns2",daemonset="ds2"} 0
+				kube_daemonset_status_number_unavailable{daemonset="ds1",namespace="ns1"} 0
+		    kube_daemonset_status_number_unavailable{daemonset="ds2",namespace="ns2"} 0
```

Comment:
We should check for a none zero unavailable value. And also mix using of blank and tab.